### PR TITLE
SmartSystemMenu: Add version 2.18.0

### PR DESCRIPTION
**Pull request by hu3rror**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

No persist file.

### CODE REVIEW

**Review by niheaven — APPROVED** [review]

File: bucket/smartsystemmenu.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "license": "MIT",
+    "url": "https://github.com/AlexanderPro/SmartSystemMenu/releases/download/v2.19.0/SmartSystemMenu_v2.19.0.zip",
+    "hash": "c83980cfe4a011d9defb42ecbc3e54249f6186293735e369dcf10e5fb154de21",
+    "shortcuts": [
+        [
+            "SmartSystemMenu.exe",
+            "SmartSystemMenu"
+        ]
+    ],
+    "bin": "SmartSystemMenu.exe",
```
Comment (rashil2000) [inline bucket/smartsystemmenu.json:14]: Commandline arguments?

Comment (niheaven) [reply bucket/smartsystemmenu.json:14]: Yes, it has cli, https://github.com/AlexanderPro/SmartSystemMenu#command-line-interface.

Comment (hu3rror) [reply bucket/smartsystemmenu.json:14]: > Yes, it has cli, https://github.com/AlexanderPro/SmartSystemMenu#command-line-interface.

Thanks for explaining. I didn't realize I had to explain XD

**Pull request merged**